spec/javascripts/mobile/mobile_comments_spec.js

```diff
@@ -155,6 +155,18 @@ describe("Diaspora.Mobile.Comments", function(){
       expect(this.toggleReactionsLink.text().trim()).toBe("1 comment");
       expect(this.toggleReactionsLink.attr("href")).toBe("/posts/" + postGuid + "/comments.mobile");
     });
+
+    it("Creates the reaction link when there are no reactions (french locale)", function() {
+      var parent = this.toggleReactionsLink.parent();
+      var postGuid = this.bottomBar.parents(".stream-element").data("guid");
+      this.toggleReactionsLink.remove();
+      parent.prepend($("<span/>", {"class": "show-comments"}).text("Aucun commentaire"));
+
+      Diaspora.Mobile.Comments.increaseReactionCount(this.bottomBar);
+      this.toggleReactionsLink = this.bottomBar.find(".show-comments").first();
+      expect(this.toggleReactionsLink.text().trim()).toBe("1 comment");
```

Comment:
The code changing the comments count tries to get the number from the bottom bar, increases the number and sets the new text afterwards. In the English locale the text is “0 comments”, so it finds the “0” and is able to parse and increase it. For the French locale you have “Aucun commentaire”, so finding the number and parsing it failed. This PR changes the code to use 0 if it's unable to find a number.

To test the correct behavior I have to set some text that doesn't contain a number and since the issue has been reported for the French locale I used “Aucun commentaire”. Without the change in `app/assets/javascripts/mobile/mobile_comments.js` the test fails.